packages/tool-cache/src/tool-cache.ts

```diff
@@ -551,7 +551,7 @@ export async function getManifestFromRepo(
   owner: string,
   repo: string,
   auth?: string,
-  branch = 'master'
+  branch = 'main'
```

Comment:
check w/ @bryanmacfarlane

Reply:
Going to pull this change out and address it in another issue. We should probably default to the default branch of a repo, or force a user to specify, but I don't want to make breaking changes here right now.